ee/clickhouse/queries/test/__snapshots__/test_retention.ambr

```diff
@@ -656,6 +656,174 @@
            intervals_from_base
   '
 ---
+# name: TestClickhouseRetention.test_groups_filtering_person_on_events_v2
+  '
+  WITH actor_query AS
+    (WITH 'Week' as period,
+          NULL as breakdown_values_filter,
+          NULL as selected_interval,
+          returning_event_query as
+       (SELECT toStartOfWeek(toTimeZone(toDateTime(e.timestamp, 'UTC'), 'UTC')) AS event_date,
+               if(notEmpty(overrides.person_id), overrides.person_id, e.person_id) AS target
+        FROM events e
+        LEFT OUTER JOIN
```

Comment:
so @neilkakkar the events query is what coordinates this

Reply:
`retention_event_query` does, but `event_query.py` does not need the distinction?

Reply:
`event_query.py` just has the JOIN information, but from what I'm seeing, `retention_event_query.py` chooses which one.

Would make sense though if the `event_query` was choosing v1 or v2, not sure if that's better

Reply:
right so you just mean let's do everything at the `event_query` level?